src/db/add_package.rs

```diff
@@ -228,35 +233,66 @@ pub(crate) async fn add_doc_coverage(
 
 /// Adds a build into database
 #[instrument(skip(conn))]
-pub(crate) async fn add_build_into_database(
+pub(crate) async fn finish_build(
     conn: &mut sqlx::PgConnection,
-    release_id: i32,
+    build_id: i32,
```

Comment:
Thinking out loud but maybe we should add newtypes for things like `build_id` (named `BuildId(i32)`), `crate_id`, `release_id`, etc to make it more obvious what we're manipulating and also to prevent trying to pass any `i32` value. This is just to make code more clear, so not a blocker.

Reply:
I like the idea, I feel like I would like to do this in a separate PR

Reply:
Yes absolutely. Let's just open an issue so it's not forgotten.